From a751428d7d268cf548b9b6a9440cf59bce79120a Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Tue, 21 May 2013 18:03:17 +0000
Subject: [PATCH 5/6] update the rules file

---
 debian/rules.em |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index cd3d54b..f12cd0e 100644
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -34,7 +34,7 @@ override_dh_auto_build:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_build
+	dh_auto_build && make docs
 
 override_dh_auto_test:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
1.7.9.5

