From 55954e333e832d179388c5a411ba61f0b6811f01 Mon Sep 17 00:00:00 2001
From: Vincent Rabaud <vrabaud@willowgarage.com>
Date: Fri, 24 May 2013 06:32:56 +0000
Subject: [PATCH 6/6] provide fixed rules.em file

---
 debian/rules.em |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index f12cd0e..8dd351b 100644
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -34,7 +34,7 @@ override_dh_auto_build:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_build && make docs
+	dh_auto_build && cd obj-* && make docs
 
 override_dh_auto_test:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
1.7.9.5

